cmd/sonobuoy/app/args.go

```diff
@@ -348,6 +351,11 @@ func AddPluginEnvFlag(p *PluginEnvVars, flags *pflag.FlagSet) {
 	flags.Var(p, "plugin-env", "Set env vars on plugins. Values can be given multiple times and are in the form plugin.env=value")
 }
 
+// AddPluginListFlag adds the flag to keep track of which built-in plugins to use.
+func AddPluginListFlag(p *[]string, flags *pflag.FlagSet) {
+	flags.StringSliceVarP(p, "plugin", "p", []string{"e2e"}, "Describe which plugin's images to interact (Valid plugins are 'e2e', 'systemd-logs').")
```

Comment:
Should this default to both plugins now? Seems like that would be simpler UX here.

Reply:
Yes it looks like this is just used for the images command. If we are going to, by default, try to run both of those plugins and the `images` commands can support both of them, I agree it should.